src/operation/edit.rs

```diff
@@ -52,7 +52,7 @@ pub trait Editable {
     /// <div>
     ///     <div class="recommend"></div>
     ///     <div class="results"></div>
-    ///     <div class="ad"></div>
```

Comment:
The reason I did this is because many code editors trim extra spaces from the ends of strings, including mine (VS Code). To better illustrate this, I'll replace the relevant spaces with `.`:
```html
    /// let html = r#"
    ///.<div>
    ///.....<div class="recommend"></div>
    ///.....<div class="results"></div>
    ///.....<div class="ad"></div>
    ///.</div>"#;
```

Then the result of the operation is:

```html
    /// assert_eq!(html, r#"
    ///.<div>
    ///.....<div class="recommend"></div>
    ///.....<div class="results"></div>
    ///....
    ///.</div>"#)
```

However, when my editor saves the file, it automatically saves to this:

```html
    /// assert_eq!(html, r#"
    ///.<div>
    ///.....<div class="recommend"></div>
    ///.....<div class="results"></div>
    ///
    ///.</div>"#)
```

Which actually makes it a different string and makes the test fail.

This is why I added another element afterwards, so that the line doesn't have spaces after.

Anyway, I'm not sure how you'd prefer addressing this, if at all, I could use a different editor to change it back. But yeah, up to you.

Reply:
I am using VSCode (with rust-analyzer), too, but do not have this problem in `///` blocks after "Format Document". So I think the old example is enough.

Reply:
I've reverted it